### PR TITLE
added routes & responsive nav

### PR DESCRIPTION
**Pull request by Muniba-Ahmed**

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+            </Box>
+          </Toolbar>
+        </Container>
+      </AppBar>
+    </div>
+  );
+}
+
+export default Navbar;
+////////////////////////////////////////////////////////////
```
Comment (AdnanCodes) [inline src/components/Navbar.js:172]: random lines of comments?

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  Container,
+  Avatar,
+  Button,
+  Tooltip,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import AdbIcon from "@mui/icons-material/Adb";
+
+// const pages = ['Products', 'Pricing', 'Blog'];
+const pages = ["Home", "Menu", "Contact"];
```
Comment (AdnanCodes) [inline src/components/Navbar.js:20]: Clicking on the button doesn't update the URL, so it isn't attempting the change the page.

**Review by AdnanCodes — CHANGES_REQUESTED** [review]
Just two things to be looked. 

Great use of Material UI.

File: src/components/Navbar.js (hunk 1 above)
Comment (Muniba-Ahmed) [reply src/components/Navbar.js]: its now fixed.

File: src/components/Navbar.js (hunk 2 above)
Comment (Muniba-Ahmed) [reply src/components/Navbar.js:18]: I've removed this import and its lines of code.

**Review by AdnanCodes — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js (hunk 2 above)
Comment (Muniba-Ahmed) [reply src/components/Navbar.js:18]: Added styles to links.

File: src/App.js [hunk 3]
```diff
@@ -1,30 +1,22 @@
...
-        <Stack spacing={2} direction="row">
-          <Button variant="text">Text</Button>
-          <Button variant="contained">Contained</Button>
-          <Button variant="outlined">Outlined</Button>
-        </Stack>
-      </header>
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
```
Comment (AdnanCodes) [inline src/App.js:15]: I just realized there would be a difference between a landing page and the home page or not.

if that's the case then we home page link can go away as the donut shop logo serves as the home and we need to add `exact` prop so the home doesn't load accidentally on other pages, unless react-router has updated this behavior 

As you can see clicking home, binds home to URL 
image

In addition, clicking other nav buttons, URL should update to page route but it should not be capital, everything is lowercase

**Review by AdnanCodes — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+                sx={{
+                  display: { xs: "block", md: "none" },
+                }}
+              >
+                {pages.map((page) => (
+                  <MenuItem key={page} onClick={handleCloseNavMenu}>
+                    <Typography textAlign="center">
+                      <Link
+                        style={{ textDecoration: "none", color: "black" }}
+                        to={`/${page}`}
```
Comment (AdnanCodes) [inline src/components/Navbar.js:97]: `page` is being passed from your array `pages` which has all caps.

you can add `/${page.toLowercase()}`

**Review by AdnanCodes — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js (hunk 4 above)
Comment (Muniba-Ahmed) [reply src/components/Navbar.js:97]: The routes are showing in lowercase now.

File: src/App.js (hunk 3 above)
Comment (Muniba-Ahmed) [reply src/App.js:15]: I've removed the home button and now the logo is routing to the home/landing page.

**Review by AdnanCodes — APPROVED** [review]

**Pull request merged**